meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-support/stm32-mfgtool-files/stm32-mfgtool-files/provision.sh.in

```diff
@@ -344,7 +344,7 @@ if [ "${need_reboot}" -eq "1" ]; then
 	fastboot reboot
 	sleep 2
 	# use TSV with usb entries only, as we don't want to re-flash binaries to eMMC again
-	STM32_Programmer_CLI -c port="${hwPort}" -d stm32-mfgtool-files/FlashLayout_emmc_@@BOARD_NAME@@-optee.tsv
+	STM32_Programmer_CLI -c port="${hwPort}" -d stm32-mfgtool-files/FlashLayout_stm32mp1-usb.tsv
```

Comment:
Could we avoid using the same literal in `lmp-mfgtool-machine-custom.inc` and in `provision.sh.in`?

like:
```
STM32MP1_USB_FLASHLAYOUT = "FlashLayout_stm32mp1-usb.tsv"
LMP_FLASHLAYOUT:stm32mp1common = ${STM32MP1_USB_FLASHLAYOUT}
...
STM32_Programmer_CLI -c port="${hwPort}" -d stm32-mfgtool-files/@@FLASHLAYOUT_USB@@
...
sed ...
```

Reply:
fixed, thanks!